torch/nn/quantized/dynamic/modules/rnn.py

```diff
@@ -113,6 +113,23 @@ def process_weights(ihhh, layer, suffix, qweight, bias, dtype):
                     self._all_weight_names.extend([ih_name, hh_name])
                     self._all_weight_values.extend([ih, hh])
 
+    def _get_name(self):
+        return 'DynamicQuantizedRNN'
+
+    def extra_repr(self):
+        s = '{input_size}, {hidden_size}'
+        if self.num_layers != 1:
+            s += ', num_layers={num_layers}'
+        if self.bias is not True:
+            s += ', bias={bias}'
+        if self.batch_first is not False:
+            s += ', batch_first={batch_first}'
+        if self.dropout != 0:
+            s += ', dropout={dropout}'
+        if self.bidirectional is not False:
+            s += ', bidirectional={bidirectional}'
+        return s.format(**self.__dict__)
```

Comment:
hey @jamesr66a, can we avoid using `**self.__dict__` in the future? Attributes are not guaranteed to be in `__dict__` (for example, if they live in a backing `script::Module`). We should prefer to use normal attribute access, even if it's more verbose.